hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieLockConfig.java

```diff
@@ -106,7 +107,13 @@ public class HoodieLockConfig extends HoodieConfig {
       .key(FILESYSTEM_LOCK_PATH_PROP_KEY)
       .noDefaultValue()
       .sinceVersion("0.8.0")
-      .withDocumentation("For DFS based lock providers, path to store the locks under.");
+      .withDocumentation("For DFS based lock providers, path to store the locks under. If don't provide one, will use Table's meta path as default");
```

Comment:
Here we call `.noDefaultValue()`, so can we remove `If don't provide one, `, just say: `use Table's meta path as default.`?